app/src/main/java/com/simprints/intentlauncher/ui/intent/model/ResponseEvent.kt

```diff
@@ -0,0 +1,11 @@
+package com.simprints.intentlauncher.ui.intent.model
+
+
+data class ResponseEvent(
+    val id: String,
+    val type: String,
+    val scopeId: String,
+    val projectId: String,
+    val createdAtMs: Long,
+    val payload: Map<String, Any>
+)
```

Comment:
Am I the only one annoyed by the "No new line at the ned of file" warnings in git? :D